sympy/crypto/crypto.py

```diff
@@ -2246,3 +2246,77 @@ def decipher_gm(message, key):
         m <<= 1
         m += not b
     return m
+
+
+########### RaiFence Cipher #############
+
+def encipher_railfence(message,rails):
+    """
+     link : https://en.wikipedia.org/wiki/Rail_fence_cipher
+     
+     performs the Rail Fence Encryption on the plain text. 
```

Comment:
This should be the first line of the docstring. The link should be in a references section (see how other docstrings in SymPy are formatted). 

Also, the first word of a sentence should be capitalized.